src/object_based/extent.cpp

```diff
@@ -141,7 +142,7 @@ namespace ear {
   }
 
   Eigen::VectorXd SpreadingPanner::panningValuesForWeight(
-      const WeightingFunction& weightFunc) {
+      const WeightingFunction &weightFunc) {
     Eigen::VectorXd weights(_panningPositions.rows());
     for (int i = 0; i < _panningPositions.rows(); ++i) {
       weights(i) = weightFunc(_panningPositions.row(i));
```

Comment:
could you return from this function using 
```(weights.transpose() * _panningPositionResults).normalized()``` to avoid another temp? Not sure whether the intermediate is an expression or a temporary so maybe not.

Reply:
I think this is outdated, too. It now still uses a temporary, but a smaller one. Ideally I'd like to eliminate this by returning an `Eigen::Ref` (or a plain reference), but I think that's orthogonal to this PR.